src/Authentication/appsettings.Development.json

```diff
@@ -69,5 +69,8 @@
   },
   "AccessManagementSettings": {
     "ApiAccessManagementEndpoint": "https://platform.at22.altinn.cloud/accessmanagement/api/v1/"
+  },
+  "PaginationOptions": {
+    "Size": 200
```

Comment:
200?

Reply:
yes, that is what Rune and Aleksander recommended for production. During xunit testing it could be lower, but it runs through a for loop, so it is quite fast anyway. Small numbers are for manual testing